Map/MapManager.gd

```diff
@@ -0,0 +1,26 @@
+extends Node2D
+## Class to manage backend for rooms (generation and such)
+
+signal populate_floor ## call this to generate rooms in a floor
```

Comment:
is this signal really needed ? the signal to say we finished maybe, but a signal to start ? can't we just call the function ?

Edit : it won't even be needed since we won't use floors